worker/uniter/runner/jujuc/export_test.go

```diff
@@ -11,3 +11,12 @@ import (
 func HandleSettingsFile(c *RelationSetCommand, ctx *cmd.Context) error {
 	return c.handleSettingsFile(ctx)
 }
+
```

Comment:
Do we need this here? It is more of a test utility than an export for testing.

I would be inclined to do it in the test file; see my other comment.

Reply:
We need to use un-exported values in the test i.e. JujuLogCommand.ctx (notice the lowercase ctx), so if we move it here then we can test without exporting values unnecessarily for people to modify outside of the package.

Protected namespace would be useful here.